test/scripts/jenkins_unit.sh

```diff
@@ -3,4 +3,8 @@
 set -e
 source "$(dirname $0)/../../src/dev/ci_setup/setup.sh"
 
+cd packages/kbn-plugin-helpers
+yarn test
```

Comment:
We could also do `yarn kbn run test` to run tests in all packages. In https://github.com/elastic/kibana/pull/16531 I'm running the tests using Kibana's mocha setup, but maybe it should just run tests locally in the packages? Pros and cons to both approaches.

Reply:
Maybe aim towards using Kibana's Jest and Mocha setup, but allow packages to use their own setup when being pulled into the monorepo like this?